src/set_theory/game/pgame.lean

```diff
@@ -705,81 +710,90 @@ Specifically, there is a bijection between the moves for Left in `x` and in `y`,
 for Right, and under these bijections we inductively have `relabelling`s for the consequent games.
 -/
 inductive relabelling : pgame.{u} → pgame.{u} → Type (u+1)
-| mk : Π {x y : pgame} (L : x.left_moves ≃ y.left_moves) (R : x.right_moves ≃ y.right_moves),
+| mk : Π {x y : pgame} (L : x.left_moves ≃ y.left_moves) (R : y.right_moves ≃ x.right_moves),
          (∀ i, relabelling (x.move_left i) (y.move_left (L i))) →
-         (∀ j, relabelling (x.move_right (R.symm j)) (y.move_right j)) →
+         (∀ j, relabelling (x.move_right (R j)) (y.move_right j)) →
```

Comment:
The new definition of `relabelling` indeed better matches `restricted`, but as you noted on Zulip, `restricted` doesn't seem to have much use, and if we decide to remove it then it makes little sense to match it. Also swapping the two sides of `R` makes it necessary to write `.symm` in addition in four relabelling constructions (e.g. `add_zero_relabelling` and `short_relabelling`) and it's not clear it's a price worth paying.

If you just want to remove the `.symm` in the last condition (can you point to some advantages?), how about keeping `R` but changing the condition to
```suggestion
         (∀ j, relabelling (x.move_right j) (y.move_right (R j))) →
```
?

Reply:
The conditions on `mk` make reference to `R.symm` only, and not to `R` alone. That's the main motivation behind this change. 

Your suggestion seems even better, so I'm going to try it out.